rel/i18n/emqx_mgmt_api_clients.hocon

```diff
@@ -133,4 +133,495 @@ By specifying the `since` parameter, it can return the number of sessions that h
+seqno_q1_comm.label:
+"""Seqno Q1 Comm"""
+
+seqno_q1_dup.desc:
+"""Sequence number of the last QoS1 message sent to the client, that hasn't been acked (Durable sessions only)"""
+seqno_q1_dup.label:
+"""Seqno Q1 Dup"""
+
+seqno_q1_next.desc:
+"""Sequence number of the next QoS1 message to be sent to the client (Durable sessions only)"""
+seqno_q1_next.label:
+"""Seqno Q1 Next"""
+
+seqno_q2_comm.desc:
+"""Sequence number of the last PUBCOMP received from the client (Durable sessions only)"""
+seqno_q2_comm.label:
+"""Seqno Q2 Comm"""
+
+seqno_q2_dup.desc:
+"""Sequence number of the last QoS2 message sent to the client, that hasn't been acked (Durable sessions only)"""
+seqno_q2_dup.label:
+"""Seqno Q2 Dup"""
+
+seqno_q2_rec.desc:
+"""Sequence number of last PUBREC received from the client (Durable sessions only)"""
+seqno_q2_rec.label:
+"""Seqno Q2 Rec"""
+
+seqno_q2_next.desc:
+"""Sequence number of next QoS2 message to be added to the batch (Durable sessions only)"""
+seqno_q2_next.label:
+"""Seqno Q2 Next"""
+
+authz_cache_access.desc:
+"""Access type"""
+authz_cache_access.label:
+"""Access Type"""
+
+authz_cache_result.desc:
+"""Allow or deny"""
+authz_cache_result.label:
+"""Result"""
+
+authz_cache_topic.desc:
+"""Topic name"""
+authz_cache_topic.label:
+"""Topic Name"""
+
+authz_cache_updated_time.desc:
+"""Update time"""
+authz_cache_updated_time.label:
+"""Updated Time"""
+
+keepalive_interval.desc:
+"""Keepalive Interval"""
+keepalive_interval.label:
+"""Keepalive"""
+
+topic.desc:
+"""Topic name"""
+topic.label:
+"""Topic Name"""
+
+qos.desc:
+"""QoS"""
+qos.label:
+"""QoS"""
+
+nl.desc:
+"""No local"""
+nl.label:
+"""No Local"""
+
+rap.desc:
+"""Retain as published"""
+rap.label:
+"""Retain as Published"""
+
+rh.desc:
+"""Retain handling"""
+rh.label:
+"""Retain Handling"""
+
+requested_client_fields.desc:
+"""Comma separated list of client fields to return in the response"""
+requested_client_fields.label:
+"""Requested Client Fields"""
+
+payload_encoding.desc:
+"""Client's inflight/mqueue messages payload encoding."""
+payload_encoding.label:
+"""Payload Encoding"""
+
+max_payload_bytes.desc:
+"""Client's inflight/mqueue messages payload limit. The total payload size of all messages in the response will not exceed this value. Messages beyond the limit will be silently omitted in the response. The only exception to this rule is when the first message payload is already larger than the limit, in this case, the first message will be returned in the response."""
+max_payload_bytes.label:
+"""Max Payload Bytes"""
+
+proto_ver.desc:
+"""Protocol version"""
+proto_ver.label:
+"""Protocol version"""
+
+like_clientid.desc:
+"""Fuzzy search `clientid` as substring"""
+like_clientid.label:
+"""Fuzzy search `clientid`"""
+
+like_username.desc:
+"""Fuzzy search `username` as substring"""
+like_username.label:
+"""Fuzzy search `username`"""
+
+gte_created_at.desc:
```

Comment:
Nit: this field already exists above, and some of the fields around this one are also duplicated.